app/src/org/commcare/android/tasks/FormRecordLoaderTask.java

```diff
@@ -238,7 +238,7 @@ protected void onPostExecute(Integer result) {
      * @see android.os.AsyncTask#onProgressUpdate(Progress[])
      */
     @Override
-    protected void onProgressUpdate(Pair<Integer, ArrayList<String>>... values) {
+    protected void onProgressUpdate(Pair<FormRecord, ArrayList<String>>... values) {

```

Comment:
Changed Integer (FormRecord ID) argument to the actual FormRecord so that IncompleteFormListAdapter.notifyPriorityLoaded can incrementally add the record to its list of current forms (if it satisfies the current query).